content/en/localstack/configuration.md

```diff
@@ -168,6 +168,37 @@ Some of the services can be configured to switch to a particular provider:
 | `KMS_PROVIDER` |  `moto` (default) and `local-kms` |
 | `SQS_PROVIDER` |  `moto` (default) and `elasticmq` |
 
+## Profiles
+
+LocalStack support configuration profiles which is stored in the `~/.localstack` config directory. A configuration profile is a set of environment variables stored in an `.env` file in the LocalStack config directory. Here is an example of what configuration profiles might look like:
+
+```sh
+% tree ~/.localstack 
+/home/username/.localstack
+├── default.env
+├── dev.env
+└── pro.env
+```
+
+Let's say you wish to have a look at the `dev.env` profile. You can do so by pushing the following command:
+
+```sh
+% cat ~/.localstack/dev.env
+DEBUG=1
+DEVELOP=1
+```
+
+You can load a profile by either setting the `env` variable `CONFIG_PROFILE=<profile>` or the `--profile=<profile>` CLI flag when using the CLI. Let's take an example to load the `dev.env` profile file if it exists:
+
+{{< command >}}
+python -m localstack.cli.main --profile=dev start --host
+{{< / command >}}
+
+If no profile is specified, the `default.env` profile will be loaded. While explicitly specified, the environment variables will always overwrite the profile. To display the config environment variables, you can use the following command:
```

Comment:
just for future docs: having each sentence in a separate line helps greatly for viewing git diffs for later changes